src/utils/ip.ts

```diff
@@ -23,6 +23,10 @@
  **********************************************************************************/
 
 import { networkInterfaces } from 'os'
+import axios from 'axios'
+import { log } from './pino'
+
+export let publicIpAdress = ''
```

Comment:
A little bit typo supposed to be: `publicIpAddress` or `publicIPAddress`

Reply:
nice catch